src/utils/perfherder.js

```diff
@@ -131,15 +138,14 @@ const prepareData = async (subtestsInfo) => {
     };
   });
 
-  return {
-    // Link to Perfherder with all subtests
-    perfherderUrl: perherderGraphUrl(signatureIds),
-    data: subtestsPerfData,
-  };
+  return data;
```

Comment:
The refactor of this function is to help me when I produce data for `test/expected`.